db.py

```diff
@@ -0,0 +1,145 @@
+"""
+This is the database for the webapp ReeDistribute
+All information collected by this program remains anonymous
+Everything that is stored is either for functionality or statistics to prove the site's valaccount_idity
+"""
+
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, Float, String, Boolean
+from sqlalchemy import create_engine
+
+Base = declarative_base() #Make it an OOP
+engine = create_engine('postgresql://@localhost:5432/myDB', echo = True)
+
+class Account(Base):
+    """
+    The main user class for ReeDistribute Login.abs
+    Attributes: account_id (int), name (str), email (str), consumer_key (str), anon (int), admin (int)
+    """
+    __tablename__='accounts' #Store all account objects in accounts
+
+    account_id = Column(Integer, primary_key=True, nullable = False) #The key value for account account_identification

```

Comment:
Nope, I haven't yet written that part yet, but soon to come as methods in the donor and student classes or as a normal python function. I will be testing to figure out which one is better/functions at all.